ansible/roles/aws-kubectl-config/tasks/packages.yml

```diff
@@ -0,0 +1,128 @@
+---
+ Used for older versions of amazon linux
+- name: "Enable epel repo"
+  yum_repository:
+    name: epel
+    description: "Extra Packages for Enterprise Linux 6 - $basearch"
+    enabled: 1
+    mirrorlist: "https://mirrors.fedoraproject.org/metalink?repo=epel-6&arch=$basearch"
+    gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-6"
+    gpgcheck: 1
+    failovermethod: priority
+  become: yes
+  when: is_yum
+  register: epel_enabled
+  tags:
+    - packages
+
+- name: "Install package dependencies - apt"
+  become: true
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - unzip
```

Comment:
We added git elsewhere. it seems that here and below (yum) would be the place to include it. Could you please add it?